deeplabcut/pose_estimation_tensorflow/datasets/pose_multianimal_imgaug.py

```diff
@@ -175,6 +179,12 @@ def build_augmentation_pipeline(self, apply_prob=0.5):
         pipeline = iaa.Sequential(random_order=False)
 
         pre_resize = cfg.get("pre_resize")
+
+        if cfg.get("traintime_resize", False):
+            # let's hard code it
```

Comment:
Why? This could precisely be the value of `traintime_resize`. How is this different from `pre_resize` though?

Reply:
I set this True for datasets in which images have super large resolution. To save efforts, I hardcoded it to 400 so I don't have to tune parameters and the same resizing will be applied at inference time. No much difference from pre_resize except I kept the aspect ratio